templates/blocks/footer.html

```diff
@@ -1,6 +1,16 @@
 {% extends "dsfr/footer.html" %}
 {% load menu_tags %}
 
+{% block brand %}
+  <p class="fr-logo" title="{{ settings.content_manager.CmsDsfrConfig.footer_brand|default:'république française' }}">
+    {{ settings.content_manager.CmsDsfrConfig.footer_brand_html| default_if_none:'république<br />française' | safe }}
```

Comment:
Apparemment il faut une majuscule à République, et aussi à Française : 
```suggestion
    {{ settings.content_manager.CmsDsfrConfig.footer_brand_html| default_if_none:'République<br />Française' | safe }}
```

Reply:
Du coup j'ai plutôt 
- Repris la formulation "Intitulé officiel" donnée en exemple dans https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/en-tete
- Ajouté un lien vers la charte sur la marque de l'État donnant les divers intitulés possibles https://www.gouvernement.fr/charte/charte-graphique-les-fondamentaux/le-bloc-marque